arkouda/strings.py

```diff
@@ -754,3 +754,16 @@ def save(self, prefix_path : str, dataset : str='strings_array',
         """       
         self.bytes.save(prefix_path=prefix_path, 
                                     dataset='{}/values'.format(dataset), mode=mode)
+
+    def register(self, user_defined_name : str) -> 'Strings':
+        return Strings(self.offsets.register(user_defined_name+'_offsets'),
+                       self.bytes.register(user_defined_name+'_bytes'))
+
+    def unregister(self) -> None:
+        self.offsets.unregister()
+        self.bytes.unregister()
+
+    def attach(user_defined_name : str) -> 'Strings':
```

Comment:
this is probably a `@classmethod`

Reply:
@mhmerrill this interesting. I always thought a class method had to be decorated with @classmethod. But, this does indeed work. However, you cannot call self from inside this method (analogous method below):

![image](https://user-images.githubusercontent.com/10785153/96778419-ae5add00-13b9-11eb-8617-a0ca4b374baf.png)


![image](https://user-images.githubusercontent.com/10785153/96778389-a26f1b00-13b9-11eb-8cf7-54cc15ab8a70.png)